client/components/dangerousHtml/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+export default React.createClass( {
+	propTypes: {
+		tag: React.PropTypes.string.isRequired,
+		html: React.PropTypes.string.isRequired,
+		className: React.PropTypes.string,
+	},
+
+	componentDidMount() {
+		ReactDom.findDOMNode( this ).innerHTML = this.props.html;
+	},
+
+	componentDidUpdate( prevProps ) {
+		if ( prevProps.html !== this.props.html ) {
+			this.componentDidMount();
```

Comment:
We shouldn't call a lifecycle method like this. Instead, extract the logic to a function and call it from both places:

```jsx
componentDidMount() {
	this.updateHtml();
},

componentDidUpdate( prevProps ) {
	if ( prevProps.html !== this.props.html ) {
		this.updateHtml();
	}
},

updateHtml() {
	ReactDom.findDOMNode( this ).innerHTML = this.props.html;
}
```